docs/superpowers/specs/2026-05-16-issue-creation-dialog-design.md

```
@@ -0,0 +1,435 @@
+
+- Command-palette entry point, sidebar entry point, and "convert chat
+  selection to issue" entry point.
+- Post-creation actions on existing issues (create worktree button on
+  `IssueDetail`, create branch, link existing PR/branch, add comment) —
+  Phase 2.
+- Multi-provider implementations beyond GitHub — Phase 3.
+- Editing or closing existing issues from inside Ryco.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)